### ISSUE TITLE
Godot 4 - object is class doesn't work

### ISSUE THREAD

**Issue by BastiaanOlij** [HungryProton/scatter#112]
In Godot 4 `if object is Class` doesn't work as it did in Godot 3 causing `modifier_stack_plugin.gd` to fail.

A dirty workaround is:
```
func _can_handle(object):
	var script : Script = object.get_script()
	if !script:
		return false
	
	return script.resource_path == "res://addons/proton_scatter/src/scatter.gd"
```

It may be possible the original syntax works if you use `class_name ProtonScatter` in `scatter.gd`, you no longer need to use `add_custom_type`, registration of the classes happens automatically. You can use `@icon` to give the class an icon.

I haven't tried this because it would require more major changes to the source.

**Comment by HungryProton** [comment]
Thanks for the report, should be fixed in a460a60c6b2e139ecf4842b455969785506e4dac

The function `is_instance_of()` takes care of that now. 

(The lack of class_name across the plugin is mostly because of the old guidelines from Godot 3. It avoids situations where the plugin is only half working because the user forgot to enable it in the project settings)

**Comment by HungryProton** [comment]
Come to think about it, this shouldn't be an issue since everything compared against is a const expression. Which version of Godot are you using?
